ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -500,6 +502,13 @@
   @Default("false")
   public final boolean routerCrossColoRequestToDcWithMostReplicas;
 
+  /**
+   * The maximum number of outgoing delete operations in background deleter. 0 means no limit.
+   */
+  @Config(ROUTER_BACKGROUND_DELETER_MAX_CONCURRENT_OPERATIONS)
+  @Default("0")
```

Comment:
Prefer to use -1. To reduce some `<=` confusion in below.

Reply:
I prefer 0, because of the getIntegerInRange method. If I set the range from -1 to MAX_INT, then 0 would be another special value besides -1. Setting range from 0 to MAX_INT solve this issue.

Reply:
got you. Sounds good.